content/en/docs/setup/production-environment/tools/kubeadm/dual-stack-support.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Dual-stack support with kubeadm
```

Comment:
The improvement is dual stack support, but once that lands it's just “dual stack” (think about the page about RBAC; once it was new, but the page about it doesn't talk about RBAC support, just about RBAC).

How about:
```suggestion
title: Enable dual-stack networking using kubeadm
```
?

Reply:
@neolit123 Do you agree to change the title?

`Dual-stack support with kubeadm` is simple.
I prefer `Enable dual-stack networking using kubeadm` as Tim says. The page mainly tells about how to enable it.

Reply:
i don't object.